puppet/zulip/manifests/postgres_appdb_base.pp

```diff
@@ -12,10 +12,21 @@
         # Needed for our full text search system
         "${postgresql}-tsearch-extras",
       ]
-      zulip::safepackage {
+      $debian_release_version_array = scanf("${::operatingsystemrelease} ", '%d')
+      $debian_release_version = $debian_release_version_array[0]
+      if $debian_release_version < 10 {
```

Comment:
A test like `$zulip::base::release_name == 'stretch'` would be more direct, though.

Reply:
agreed

Reply:
Because of the
```
/^7.[0-9]*/  => 'wheezy',
/^8.[0-9]*/  => 'jessie',
/^9.[0-9]*/  => 'stretch',
/^10.[0-9]*/ => 'buster',
```
part, I was under the impression that we wanted to give some kind of partial support to older versions of Debian too, but otherwise, yes, I also agree that this (release name) convention is better too. Come to think of it, why are `wheezy` and `jessie` even mentioned in `puppet/zulip/manifests/base.pp` in the first place?